app/screens/custom_status/custom_status_suggestion.tsx

```diff
@@ -74,7 +74,9 @@ const CustomStatusSuggestion = ({handleSuggestionClick, emoji, text, theme, sepa
                                 textStyle={style.customStatusText}
                             />
                         </View>
-                        {Boolean(duration && isExpirySupported) && (
+                        {Boolean(duration &&
```

Comment:
Small change :) Can we move the condition out of JSX and use the variable for readability? Thank you!
```
const showCustomStatus = Boolean(duration && duration !== CustomStatusDuration.DATE_AND_TIME && isExpirySupported);
```

Reply:
Sure